src/v2/view-builder/internals/BaseForm.js

```diff
@@ -62,14 +62,12 @@ export default Form.extend({
     }
   },
 
-  addCallouts () {
-    const warningMsgs = this.options.appState.get('messages');
-    if (warningMsgs && warningMsgs.value.length) {
-      const messageCallout = createCallout({
-        content: warningMsgs.value[0].message,
-        type: 'warning',
-      });
-      this.add(messageCallout, '.o-form-error-container');
+  showMessages () {
+    // render messages as text
+    const messagesObj = this.options.appState.get('messages');
+    if (messagesObj && messagesObj.value.length && messagesObj.value[0].class === 'INFO') {
```

Comment:
- probably don't have to check `class` here?
- why only display first message instead of listing all given this is generic handler?

Reply:
I added class check to be sure we dont render error as a text, but no other reason. 
I didnt think we would have a case for multiple messages. Should we just append and generate one single string and render it ?

Reply:
Given this is generic handler, we need to account for ERROR or no `class` type.
The easiest way is probably list all messages per line (with error renders in error callout)
Concat non-error into one string is also fine.

Can create JIRA and tackle separately for `error` type.

Maybe we really don't have to care `type` now given `ERROR` is probably handled in form error handler (when none 200 response). 
So probably just 
- either list the messages
- or concat into one string

Reply:
Sounds good, I can concat into one string. I also think error case wont hit here because its going to be a non 200 response which we handle elsewhere.